### ISSUE TITLE
Export to CSV/Excel file

### ISSUE THREAD

**Issue by Isuru-Nanayakkara** [AceFire6/ordered-arrowverse#110]
Hi,

It'll be great if there was an option to export the list to CSV and/or Excel file.

**Comment by AceFire6** [comment]
Hey @Isuru-Nanayakkara. Thanks for the suggestion! That's probably something I could add.

(Sorry for the delayed response, life has been busy)

**Comment by allcoast** [comment]
Yes please. I would like this as well. I can not figure out how to format it correctly for excel format.

**Comment by neil-menghani** [comment]
I can work on this issue tomorrow

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
